content/rust/docker/rust-docker-arm.md

```diff
@@ -12,3 +12,365 @@ tags:
 - docker
 - arm
 ---
+
+This guide will walk you through the steps required to build and publish arm64 Docker images for Rust projects. This guide assumes you have a basic understanding of Rust and Docker, and that you have Rust and Docker installed.
```

Comment:
Please keep lines to 72 characters or less wide - the Markdown linter will complain about them.

Reply:
The line you have selected is basicly the same as in the "template". If that works without problems all should be fine? Anyway I moved every sentence in a new line to make long lines shorter.

Reply:
Thanks!